### ISSUE TITLE
Stream Cells - Game name & stream title merged into same line sometimes

### ISSUE THREAD

**Issue by ksnovak** [ksnovak/Multigame_Browser#86]
This happens if both elements combined don't take up more space than one row.
Need to just set `display: block;` probably.

![chrome_2018-09-27_10-56-37](https://user-images.githubusercontent.com/4551533/46164895-0c04b580-c244-11e8-858d-d85df6fcf8ae.png)

**Comment by ksnovak** [comment]
This is weirder than expected. Caused by using the bootstrap grid system.
I tried quickly setting to `display: block`, as well as adding in a `<br />`, and even changing them from `span` to `div`. In all cases, nothing was fixed.
I tried changing to:
```
<span className="streamGame">{game ? ` playing ${game.name}` : ''}</span>
<br />
ff
<span className="streamTitle">{stream.title}</span>
```

and that made things just look... weird. But that made me realize it's about the grid system instead of anything else.
